superchain/configs/configs.json

```diff
@@ -7,8 +7,10 @@
         "L1": {
           "ChainID": 1,
           "PublicRPC": "https://ethereum-rpc.publicnode.com",
-          "Explorer": "https://etherscan.io"
+          "Explorer": "https://etherscan.io",
+          "Versions": null
```

Comment:
This doesn't look quite right, do we need to add an `omitempty` struct tag somewhere?

Reply:
Scratch the above. Actually I should have remove the `Versions` field because it is now in `validation/standard`